refapp/SKWebApi/Constants.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+namespace SKWebApi;
+
+internal static class Constants
+{
+    public const string NativeSkillsDirectory = "NativeSkillsDirectory";
```

Comment:
This is a little unclear at first look. Could we rename the member to `NativeSkillsDirectoryConfigKey`, or something like that? Same for SemanticSkillsDirectory.

Reply:
Done